dashboard/client/src/pages/overview/OverviewPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { createStyles, makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import EventTable from "../../components/EventTable";
+import { MainNavPageInfo } from "../layout/mainNavContext";
+import { Metrics } from "../metrics";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      padding: theme.spacing(3),
+      backgroundColor: "white",
+    },
+    metricsContainer: {
+      marginTop: theme.spacing(2),
+    },
+  }),
+);
+
+export const OverviewPage = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <MainNavPageInfo
+        pageInfo={{ title: "Overview", id: "overview", path: "/new/overview" }}
+      />
+      <div>
+        <Typography variant="h1">Events</Typography>
+        <EventTable />
+      </div>
+      <div className={classes.metricsContainer}>
+        <Typography variant="h1">Node metrics</Typography>
```

Comment:
1. In the mock, we only have metrics for nodes in this section and that's why it's called "Node metrics".
2. We won't have job-scoped metrics soon so it doesn't make sense to simply move task/actor related metrics into job detail pages.

Combing 1 and 2, instead of changing it to "Metrics", we should start to have groupings of metrics.

The overview page might look like this:
Row 1: [cluster utilization metrics] [node count] [Recent jobs]
Row 2: [Events, filtered to show "Autoscaler", "cluster lifecycle", "job" events only]
Row 3: [Node metrics]
Row 4: [Task metrics]
Row 5: [Actor metrics]
Row 6: [Scheduler metrics]